tensorflow/python/ops/control_flow_ops_test.py

```diff
@@ -448,6 +449,19 @@ def testCond_gradient(self):
     self.assertEqual(None if context.executing_eagerly() else 0.,
                      self.evaluate(grads_false[0]))
 
+  def testCondWithGroupAndSummaries(self):
+    ops.disable_eager_execution()
```

Comment:
I don't think this line should be necessary. Since you explicitly create the graph, everything within that scope is in graph mode, not eager mode.

Reply:
gotcha. removed the line.